### DISCUSSION TITLE
2022/02/19/Hexo-Butterfly-%E5%BB%BA%E7%AB%99%E6%8C%87%E5%8D%97%EF%BC%88%E5%9B%9B%EF%BC%89Twikoo-%E8%AF%84%E8%AE%BA%E7%B3%BB%E7%BB%9F/

### DISCUSSION THREAD

**Discussion by giscus** [nickxudotme/nickxudotme.github.io#10] [General]
# 2022/02/19/Hexo-Butterfly-%E5%BB%BA%E7%AB%99%E6%8C%87%E5%8D%97%EF%BC%88%E5%9B%9B%EF%BC%89Twikoo-%E8%AF%84%E8%AE%BA%E7%B3%BB%E7%BB%9F/

本篇内容主要源于：  Twikoo 官方文档 Butterfly 安裝文檔(四) 主題配置-2   更新记录2022-04-09 更正了一个小错误，增加了图片上传功能教程与更新教程 2022-08-11 根据官网更新了 Vercel 的部署链接，很抱歉今天才发现链接从 dev 分支变成了 main

https://nickxu.me/2022/02/19/Hexo-Butterfly-%E5%BB%BA%E7%AB%99%E6%8C%87%E5%8D%97%EF%BC%88%E5%9B%9B%EF%BC%89Twikoo-%E8%AF%84%E8%AE%BA%E7%B3%BB%E7%BB%9F/

**Comment by lestil** [comment]
哈哈哈

**Comment by huochairener** [comment]
可以👍

**Comment by ikun09** [comment]
怎么换回GitHub了

**Reply by ikun09** [reply]
刚看到有个切换按钮，这个切换怎们搞得呀

**Reply by nickxudotme** [reply]
这个是，butterfly 主题自带的（）

**Reply by ikun09** [reply]
好的
